src/Core/Prefab/Globe/BuilderEllipsoidTile.js

```diff
@@ -33,24 +40,30 @@ BuilderEllipsoidTile.prototype.Prepare = function Prepare(params) {
 
     params.deltaUV1 = (st1 - start) * params.nbRow;
 
+    // transformation to align tile's normal to z axis
+    params.quatNormalToZ = new THREE.Quaternion().setFromAxisAngle(axisY, -(Math.PI * 0.5 - params.extent.center().latitude()));
+
     // let's avoid building too much temp objects
     params.projected = { longitudeRad: 0, latitudeRad: 0 };
 };
 
 // get center tile in cartesian 3D
-BuilderEllipsoidTile.prototype.Center = function Center(params) {
-    params.center = params.extent.center(this.tmp.coords[0])
+BuilderEllipsoidTile.prototype.Center = function Center(extent) {
+    return extent.center(this.tmp.coords[0])
         .as('EPSG:4978', this.tmp.coords[1]).xyz();
-    return params.center;
 };
 
 // get position 3D cartesian
 BuilderEllipsoidTile.prototype.VertexPosition = function VertexPosition(params) {
-    this.tmp.coords[0]._values[0] = params.projected.longitudeRad;
-    this.tmp.coords[0]._values[1] = params.projected.latitudeRad;
+    this.tmp.coords[0].set(
+        'EPSG:4326',
+        params.projected.longitudeRad,
+        params.projected.latitudeRad);
+
+    this.tmp.coords[0]._internalStorageUnit = UNIT.RADIAN;
 
     this.tmp.coords[0].as('EPSG:4978', this.tmp.coords[1]).xyz(this.tmp.position);
-    this.tmp.normal.copy(this.tmp.position).normalize();
+    this.tmp.normal.copy(this.tmp.coords[1].geodesicNormal);
```

Comment:
Btw, you could remove `this.tmp.normal` completely and replace VertexNormal code by `return this.tmp.coords[1].geodesicNormal`.